java/source/conf.py

```diff
@@ -0,0 +1,55 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'java-cookbook'
+copyright = '2021, apache arrow'
+author = 'apache arrow'
+
+# The full version, including alpha/beta/rc tags
+release = 'arrow cookbook'
```

Comment:
It seems other cookbooks don't have 'release' and have different values for copyright/author, can we be consistent? (Also, it's 2022 now.)

Reply:
Changed